types/src/on_chain_config/randomness_config.rs

```diff
@@ -16,8 +19,24 @@ impl AsMoveAny for ConfigOff {
     const MOVE_TYPE_NAME: &'static str = "0x1::randomness_config::ConfigOff";
 }
 
-#[derive(Deserialize, Serialize)]
-pub struct ConfigV1 {}
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+pub struct ConfigV1 {
+    pub secrecy_threshold: FixedPoint64MoveStruct,
```

Comment:
Do we need fixed point for this? I think we only need to compare calculated numbers to these numbers.

Reply:
in move we only have fixed point